app/models/comment.rb

```diff
@@ -4,5 +4,5 @@ class Comment < ActiveRecord::Base
 
   validates :content, presence: true
 
-  delegate :first_name, :last_name, :full_name, :initials_background_color, :initials,  to: :user, prefix: true
+  delegate :user_id, :first_name, :last_name, :full_name, :initials_background_color, :initials,  to: :user, prefix: true
```

Comment:
Ta zmiana jest do cofnięcia - nie jest potrzebna. Przy okazji, jest o jedna spacja za dużo przed `to:`.